tests/envs/test_gym_conversion.py

```diff
@@ -1,15 +1,12 @@
 import warnings
 
+import gym
 import pytest
 
 import gymnasium
 from gymnasium.utils.env_checker import check_env
 from tests.envs.test_envs import CHECK_ENV_IGNORE_WARNINGS
 
-pytest.importorskip("gym")
```

Comment:
Do we maybe want to move these tests into shimmy instead? My rationale is that right now, gymnasium doesn't really interact with gym directly in any way, so it shouldn't even really be a testing dependency. The interaction goes via shimmy, so shimmy should take care of it. Similarly to how we don't have any tests for dm-env here

Reply:
True, Im happy to remove the test except for testing that the environment exist in the registry as expected